{
  "_id": {
    "$oid": "605a7ec56cf0a13ef54bc1c7"
  },
  "author_association": "MEMBER",
  "issue_url": "https://api.github.com/repos/gatsbyjs/gatsby/issues/16957",
  "updated_at": "2019-08-22T21:58:50Z",
  "performed_via_github_app": null,
  "html_url": "https://github.com/gatsbyjs/gatsby/issues/16957#issuecomment-524094086",
  "created_at": "2019-08-22T21:58:50Z",
  "id": 524094086,
  "body": "> You're right though about CircleCI, I'm not sure why I had problems there\r\n\r\nYou were probably caching `node_modules` on CircleCI, I would guess. That would cause this. \r\n\r\nAnyway, looks like this isn't a bug then. Still keeping this open for a while until the issue author and others are good! ðŸ™‚ ",
  "user": {
    "gists_url": "https://api.github.com/users/sidharthachatterjee/gists{/gist_id}",
    "repos_url": "https://api.github.com/users/sidharthachatterjee/repos",
    "following_url": "https://api.github.com/users/sidharthachatterjee/following{/other_user}",
    "starred_url": "https://api.github.com/users/sidharthachatterjee/starred{/owner}{/repo}",
    "login": "sidharthachatterjee",
    "followers_url": "https://api.github.com/users/sidharthachatterjee/followers",
    "type": "User",
    "url": "https://api.github.com/users/sidharthachatterjee",
    "subscriptions_url": "https://api.github.com/users/sidharthachatterjee/subscriptions",
    "received_events_url": "https://api.github.com/users/sidharthachatterjee/received_events",
    "avatar_url": "https://avatars.githubusercontent.com/u/7701981?v=4",
    "events_url": "https://api.github.com/users/sidharthachatterjee/events{/privacy}",
    "html_url": "https://github.com/sidharthachatterjee",
    "site_admin": false,
    "id": 7701981,
    "gravatar_id": "",
    "node_id": "MDQ6VXNlcjc3MDE5ODE=",
    "organizations_url": "https://api.github.com/users/sidharthachatterjee/orgs"
  },
  "url": "https://api.github.com/repos/gatsbyjs/gatsby/issues/comments/524094086",
  "node_id": "MDEyOklzc3VlQ29tbWVudDUyNDA5NDA4Ng=="
}